libs/libxs/Makefile

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright (c) 2016-2019  Moddable Tech, Inc.
+#
+#   This file is part of the Moddable SDK Tools.
+# 
+#   The Moddable SDK Tools is free software: you can redistribute it and/or modify
```

Comment:
Don't put a large license notice like this. Keep it short, as in https://github.com/openwrt/packages/blob/master/mail/alpine/Makefile for example.